test/CommentTester.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System.IO;
+using NUnit.Framework;
+using SchemaZen.Library;
+using SchemaZen.Library.Models;
+
+namespace SchemaZen.Tests
+{
+    [TestFixture]
+    class CommentTester {
+        private Database _db;
+        private string _comment;
+
+        [SetUp]
+        public void SetUp() {
+            _db = new Database("TestScriptTableType");
+            _db.Connection = ConfigHelper.TestDB.Replace("database=TESTDB", "database=" + _db.Name);
+            _db.ExecCreate(true);
+            _comment = Database.AutoGenerateComment;
+
+            DBHelper.ExecSql(_db.Connection, TestStrings.SetupTable0Script);
+            DBHelper.ExecSql(_db.Connection, TestStrings.SetupTable1Script);
+            DBHelper.ExecSql(_db.Connection, TestStrings.SetupTableTypeScript);
+            DBHelper.ExecSql(_db.Connection, TestStrings.SetupFKScript);
+            DBHelper.ExecSql(_db.Connection, TestStrings.SetupFuncScript);
+            DBHelper.ExecSql(_db.Connection, TestStrings.SetupProcScript);
+            DBHelper.ExecSql(_db.Connection, TestStrings.SetupRoleScript);
+            DBHelper.ExecSql(_db.Connection, TestStrings.SetupTrigScript);
+            DBHelper.ExecSql(_db.Connection, TestStrings.SetupUserScript);
+            DBHelper.ExecSql(_db.Connection, TestStrings.SetupViewScript);
+        }
+
+        [Test]
+        public void TestFilesContainComment()
+        {
+            _db.Dir = _db.Name;
+            _db.Load();
+            _db.ScriptToDir();
+
+            Assert.IsTrue(validateFirstLineIncludesComment(_db.Name +
+                "\\table_types\\" + TestStrings.TableTypeFileName, _comment));
```

Comment:
My thinking was this makes sure the comment is on top of the file, would that be unnecessary?